Tribler/Test/GUI/FakeTriblerAPI/utils/network.py

```diff
@@ -74,16 +75,17 @@ def _test_port(family, sock_type, port):
     assert sock_type in (socket.SOCK_DGRAM, socket.SOCK_STREAM), "Invalid sock_type value %s" % sock_type
     assert 0 < port <= 65535, "Invalid port value %s" % port
 
-    s = None
     try:
-        s = socket.socket(family, sock_type)
-        if sock_type == socket.SOCK_STREAM:
-            s.setsockopt(socket.SOL_SOCKET, socket.SO_LINGER, struct.pack('ii', 1, 0))
-        s.bind(('', port))
+        # In py2 socket lacks the __exit__ dunder method needed for with statements,
+        # which is where contextli.closing comes in.
```

Comment:
Again, the comment is unnecessary, as we don't support py2 anymore.

Reply:
As is `contextlib` usage, of course.
Nice trick, though :+1:

Reply:
Removed. And thanks 😄